### ISSUE TITLE
grants v2: compliance edition

### ISSUE THREAD

**Issue by owocki** [gitcoinco/web#590]
(this ticket is not ready to be worked on yet)

@Ian L. @owocki I know an important part of grant work is financial compliance that funds are spent in accordance with the with budget provided in an application... would there be a way to write that logic into the tip and bounty smart contracts.

5 replies
Ian L. [24 minutes ago]
I'm sure there is, but GitCoin is still in an MVP stage, so that kind of thing is going to be delegated out-of-band for now

Ian L. [21 minutes ago]
Think of GitCoin right now (this is necessarily inaccurate btw) as a tool to facilitate awareness + gateway the payment... i.e., the credit card processor or bank transferring grand funds doesn't do that layer of compliance in the existing model.

owocki [< 1 minute ago]
do you mind if i attach this convo to the github issue (and possible create its own github issue to track compliance protocols)?

Ian L. [< 1 minute ago]
I insist! :slightly_smiling_face:

owocki [< 1 minute ago]
>  the credit card processor or bank transferring grand funds doesn’t do that layer of compliance in the existing model.

this is true but we owe it to whomeever is going to build the compliance tool to gather data for them to use!

Skip: 
This in itself could become a solution to the oversight reviews i.e. https://blog.ecivis.com/bid/207328/best-practices-for-grantees-and-grantors-grant-compliance (edited)

Ian L. [21 minutes ago]
this level of integration / functionality deserves WAY more than a 0.2 ETH bounty... it could easily become a spoke of ConsenSys unto itself

Ian L. [17 minutes ago]
in other words... it's an *awesome* idea! There's a lot of potential there, so maybe other people already thinking about this / working on it? Or maybe not--Web3 is a wild frontier tech, rife with potential as the paradigm encounters more and more experts of various current-gen / legacy systems.

Whats your background with grants like? Is this something that you'd like to get more involved with? (edited)

**Comment by owocki** [comment]
v2 of https://github.com/gitcoinco/web/issues/380

**Comment by owocki** [comment]
whoops @razordaze not @ian :P

**Comment by skip-carlson** [comment]
I have held active roles in non-profits and humanitarian groups. I have also written successful grant proposals before.

**Comment by e-n-l** [comment]
@owocki I blame my poor branding in slack. I'm going for homonymical redemption with a change in username to @e-n-l, and claiming @razordaze as my organization. So either will work henceforth.
